boards/renesas/rcar_h3_salvatorx/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: rcar_h3_salvatorx
```

Comment:
(see comment about M3 Salvator-XS board)
H3 is also another name for r8a7795 SoC, so board should be `rcar_salvator_x`

Reply:
Fixed